packages/Uploader/src/Uploader.js

```diff
@@ -0,0 +1,219 @@
+import React from "react";
+import PropTypes from "prop-types";
+import uuidv4 from "uuid/v4";
+import stylers from "@paprika/stylers/";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import { inputFileStyles, labelStyles, containerStyles } from "./Uploader.styles";
+import { getFiles } from "./helpers";
+import ProgressBar from "./components/ProgressBar";
+import types from "./types";
+import useDragAndDropEvents from "./useDragAndDropEvents";
+import useProcessFiles from "./useProcessFiles";
+
+const oneMebibyte = 1048576;
+
+const propTypes = {
+  /**
+    Accessible message for the input[type="file"].
+  */
+  a11yText: PropTypes.string,
+  /**
+    An array of string describing the allowed file types for the uploader ex. ["image/*", ".pdf", ".doc", "docx"]
+    reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#Unique_file_type_specifiers.
+  */
+  acceptableFileTypes: PropTypes.arrayOf(PropTypes.string),
+  /**
+    When false the uploader only accept one file per upload.
+   */
+  allowMultipleFile: PropTypes.bool,
+  /**
+    children function received the following parameter which you can use to build you own UI:
+
+    FileInput,   // the input[type="file"] element ready to be consumed
+    files,       // a enhance list of the user selected files
+    hasFinished, // is true when all files have been processed
+    isDisabled,  // describe the status of the component
+    isDragLeave, // will be true if a dragOver event occurred and leave the droppable area
```

Comment:
Why couldn't `!isDraggingOver` be used for this ?

Reply:
yes we can, is just a different event, up to the author how wants to use the isDraggingOver prop

Reply:
If it's not needed, I'd suggest leaving it out of the API, less is more.  If it's really needed though, we should try come up with a name that's a bit clearer. `isDragLeave` sounds a bit awkward.